app/models/trip.rb

```diff
@@ -1,6 +1,7 @@
 class Trip < ApplicationRecord
   belongs_to :user
   belongs_to :trail
+  has_many :reviews, dependent: :destroy
```

Comment:
should be has_one :review

Reply:
edited

Reply:
Not sure why, but when I changed it to has_one :review, the trip show page throws an error